bom/runtime/pom.xml

```diff
@@ -863,6 +865,39 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-core</artifactId>
+                <version>${datastax.java.driver.version}</version>
+                <exclusions>
```

Comment:
Why do we need to exclude Tinkerpop?

Reply:
Presumably the concern here was along the lines of what was discussed in KAF-189?

Reply:
Are exclusions that I did are necessary because of the `<bannedDependencies>` specified here:
https://github.com/riptano/quarkus/blob/master/build-parent/pom.xml#L218